### DISCUSSION TITLE
小米手机提示发现恶意应用 malicious applications

### DISCUSSION THREAD

**Discussion by vandermondeDET** [2dust/v2rayNG#3276] [General]
### 疑问
请问这是什么情况？是手机的问题还是程序的疏漏？

### 实际行为
安装apk包时提示
![c910befec72059171538ee13fc0d133](https://github.com/2dust/v2rayNG/assets/156590165/72719971-d3ab-4a4b-bacf-6d5c68b8020c)

### 环境信息
Redmi k
octa-core
xiaomi hyperOS 1.0
Android 14

**Comment by fishpoaem** [comment]
你的手机HyperOS系统的问题，像我的MIUI国际版就没有这样的问题

**Comment by 2dust** [comment]
老情况了，相信小米吧。。。
